torchvision/ops/_masks_to_bounding_boxes.py

```diff
@@ -0,0 +1,26 @@
+import torch
+
+
+def masks_to_bounding_boxes(masks: torch.Tensor) -> torch.Tensor:
+    """Compute the bounding boxes around the provided masks
+    The masks should be in format [N, H, W] where N is the number of masks, (H, W) are the spatial dimensions.
+    Returns a [N, 4] tensors, with the boxes in xyxy format
```

Comment:
Would you mind writing a full docstring with a Parameters and Return section, e.g. like in https://github.com/pytorch/vision/blob/master/torchvision/ops/boxes.py


I would also suggest to move this function into the `boxes.py` file instead of creating a new file